testing/trino-benchto-benchmarks/src/test/resources/sql/presto/tpcds/partitioned/q13.plan.txt

```diff
@@ -3,24 +3,24 @@ final aggregation over ()
         remote exchange (GATHER, SINGLE, [])
             partial aggregation over ()
                 join (INNER, REPLICATED):
-                    join (INNER, PARTITIONED):
-                        remote exchange (REPARTITION, HASH, ["cd_demo_sk"])
-                            scan customer_demographics
-                        local exchange (GATHER, SINGLE, [])
-                            remote exchange (REPARTITION, HASH, ["ss_cdemo_sk"])
-                                join (INNER, REPLICATED):
+                    scan customer_demographics
```

Comment:
revert for now. i don't think this optimization should be changing the Hive plans
if it does, we will need to understand what happen (besides blindly running the benchmarks)